src/main/webapp/app/tutor-course-dashboard/tutor-participation-graph/tutor-participation-graph.component.ts

```diff
@@ -1,19 +1,20 @@
-import { Component, Input, OnChanges, OnInit, SimpleChanges } from '@angular/core';
+import { Component, Input, OnChanges, OnInit, SimpleChanges, ViewEncapsulation } from '@angular/core';
 import { TutorParticipation, TutorParticipationStatus } from 'app/entities/tutor-participation';
 import { Router } from '@angular/router';
 import * as _ from 'lodash';
-import { ExerciseType } from 'app/entities/exercise';
+import { Exercise, ExerciseType } from 'app/entities/exercise';
```

Comment:
I think this import is unused.
```suggestion
import { Exercise } from 'app/entities/exercise';
```

Reply:
Hi @jpbernius , this import is used in line 21 :)
https://github.com/ls1intum/Artemis/blob/3537803032563d99e25a9e301f20790fb4d99027/src/main/webapp/app/tutor-course-dashboard/tutor-participation-graph/tutor-participation-graph.component.ts#L21

Reply:
Oh I must have missed that. OK then this is good to go!